### PR TITLE
Package update

### PR DESCRIPTION
**Pull request by pythonhubdev**
- [x] Updated packages to latest version
- [x] Added lint rules with `analysis_options.yaml`
- [x] Updated GitHub actions
- [x] Removed containers and switched to GitHub actions for setting up Dart SDK

### CODE REVIEW

**Comment by pythonhubdev** [comment]
@vkammerer I have updated with latest PR here as requested

**Comment by pythonhubdev** [comment]
Hi @vkammerer any update on this PR?

**Comment by pythonhubdev** [comment]
@lig can you review this PR

File: analysis_options.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+include: package:flutter_lints/flutter.yaml
```
Comment (vkammerer) [inline analysis_options.yaml]: you are including `flutter_lints` here, but it's not imported in `pubspec.yaml`. Instead, you imported `lints` at https://github.com/fluttercommunity/import_sorter/pull/75/files#diff-8b7e9df87668ffa6a04b32e1769a33434999e54ae081c52e5d943c541d4c0d25R21.

This line should be updated to
```yaml
include: package:lints/recommended.yaml
```

**Review by vkammerer — COMMENTED** [review]
Thank you for your PR @pythonhubdev.

I checked out your branch locally, and I could see that the dart analyzer reported several issues with it.

I just pushed https://github.com/fluttercommunity/import_sorter/pull/75/commits/7f17800069a8b85bd1aa54581846bfa34116c131 which addresses theses issues, and makes the tests pass.

**Comment by vkammerer** [comment]
This was finally published on pub.dev, under the version `5.0.0-releasecandidate.1`

**Pull request merged**